client/src/app/navbar/navbar-app/navbar-app.component.scss

```diff
@@ -1,7 +1,36 @@
-.example-spacer {
-    flex: 1 1 auto;
+$navbar-bg: #0B2149;
+$text-color: #ffffff;
+$button-text-color: $navbar-bg;
```

Comment:
Nice job using variables for themes! Hopefully this makes it easier to change colors/settings in the future if you want